docs/multiple-users.md

```diff
@@ -0,0 +1,53 @@
+# Multiple users in GCM
+
+We're sometimes asked, "Does GCM support multiple users?" The answer is a bit complex (though ultimately, it's "yes").
+
+## Foundations: Git and Git hosts
+
+Git itself doesn't have a strong concept of "user". There's the `user.name` and `user.email` which get embedded into commit headers/trailers, but these are arbitrary strings. GCM doesn't interact with this notion of a user at all. You can put whatever you want into your `user.*` config, and nothing in GCM will change at all.
+
+Git hosting providers (like GitHub or Bitbucket) _do_ have a concept of "user". Typically it's an identity like a username or email address, plus a password or other credential to perform actions as that user. You may have guessed by now that GCM (the Git **Credential** Manager) does work with this notion of a user.
```

Comment:
Git also has the concept of "username" of a remote or credential.. the `credential.username` config is a thing, as are `protocol://userinfo@host` in URLs.

I would add another para here that Git _does_ have another (not commit object) notion of user for credentials and remotes, but these are often not used and not included by default in the web UI "copy to clone command" buttons of the various hosts (except Bitbucket!)